pkg/upgrade/drain_manager.go

```diff
@@ -112,7 +112,7 @@ func (m *DrainManagerImpl) ScheduleNodesDrain(ctx context.Context, drainConfig *
 					m.log.V(consts.LogLevelError).Error(err, "Failed to cordon node", "node", node.Name)
 					_ = m.nodeUpgradeStateProvider.ChangeNodeUpgradeState(ctx, node, UpgradeStateFailed)
 					logEventf(m.eventRecorder, node, corev1.EventTypeWarning, GetEventReason(),
-						"Failed to cordon the node, %s", err.Error())
+						fmt.Sprintf("Failed to cordon the node, %s", err.Error()))
```

Comment:
We can format in advance: Use fmt.Sprintf to format the complete log message string in advance to prevent problems when the logging function handles formatting internally.